edge-patron/src/main/resources/domain/edge-patron/features/patrons.feature

```diff
@@ -0,0 +1,34 @@
+Feature: patrons tests
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*'  }
+    * def uuid = call uuid1
+    * def firstName = call random_string
+    * def lastName = call random_string
+    * def userName = call random_string
+    * def owner = random_string
+
+
+  Scenario: Return total fees/fines regardless of fee/fine being attached to an item for a patron.
+    * def itemId = call uuid1
+    * def materialTypeId = call uuid1
+    * def materialTypeName = random_string
+    * def barcode = call random_numbers
+    * def createItemResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')
+    * def materialTypeId = call uuid1
+    * def materialTypeName = random_string
+    * def barcode = call random_numbers
+    * def itemId = call uuid1
+    * def createItemResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')
+    * def createUserResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostPatronGroupAndUser')
+    * def userId = createUserResponse.createUserRequest.id
+    * def createFineResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine')
+    * def createFineResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine')
+
+    Given path 'accounts?query=(userId=='+userId+')'
+    When method GET
+    Then status 200
+    And match response.totalRecords == 2
```

Comment:
This seems to be using the `/accounts` path which is provided by mod-feesfines. 

Given this is a test for edge-patron, is that what was intended?

Reply:
@marcjohnson-kint 
i was quite confuse about this, thats why i converted this PR into draft . i will let you know for review soon. Thankyou

Reply:
@SinghAdes Apologies, I thought a review had been requested, I didn't realise it was in draft